.github/workflows/coverage.yml

```diff
@@ -56,9 +66,20 @@ jobs:
             coverage combine .tox/*/.coverage
             coverage xml
 
+      - name: Generate code coverage
+        id: coverage
+        run: |
+          cargo llvm-cov --all-features --workspace \
+            --no-fail-fast --codecov \
+            --exclude metatensor-python \
```

Comment:
Do you also want to add `--include-ffi` here? And why is `metatensor-python` excluded here and not in the instructions?